qemu/tests/balloon_hotplug.py

```diff
@@ -66,6 +65,10 @@ def run_pm_test(pm_test, plug_type):
             err += ("\nDevice is not in qtree %ss after hotplug:\n%s"
                     % (pause, vm.monitor.info("qtree")))
 
+        vm.params["balloon"] = "balloon%d" % idx
+        vm.params["balloon_dev_devid"] = "balloon%d" % idx
+        vm.params["balloon_dev_add_bus"] = "yes"
```

Comment:
Why add these params here and delete them at line 105?

Reply:
Do adding after hotplugging and deleting after hotunplugging. Thanks!